authorization/service/getUserDevices.go

```diff
@@ -67,40 +66,28 @@ func (s *Service) GetUserDevices(request *pb.GetUserDevicesRequest, srv pb.Autho
 		if err != nil {
 			return logAndReturnError(status.Errorf(codes.InvalidArgument, "cannot add device: %v", err))
 		}
-		userID, err := parseSubFromJwtToken(token)
+		owner, err := grpc.ParseOwnerFromJwtToken(s.ownerClaim, token)
 		if err != nil {
-			log.Debugf("cannot parse user from jwt token: %v", err)
+			log.Debugf("cannot parse '%v' from jwt token: %v", s.ownerClaim, err)
 		}
-		if userID == "" {
+		if owner == "" {
 			return logAndReturnError(status.Errorf(codes.InvalidArgument, "cannot get user devices: invalid userIdsFilter"))
 		}
-		userIdsFilter = []string{userID}
-	}
-
-	deviceIdsFilter := make(map[string]bool)
-	for _, deviceID := range request.GetDeviceIdsFilter() {
-		deviceIdsFilter[deviceID] = true
+		if owner == serviceOwner {
+			return sendUserDevices(request, srv, tx.RetrieveAll)
```

Comment:
RA or RD, services using service token can fill userIdsFilter or deviceIdsFilter. Why are you ignoring values from there ?

Reply:
userIdsFilter is handled by https://github.com/plgd-dev/cloud/pull/218/files#diff-c6adc03507ee9569cc1a9e1ed0004c4b456aa337c574fcdf9e8f5bce78c94f77R64

devicesIdsFilter is handled by https://github.com/plgd-dev/cloud/pull/218/files#diff-c6adc03507ee9569cc1a9e1ed0004c4b456aa337c574fcdf9e8f5bce78c94f77R33